src/components/InstanceMonitoring/_instance-monitoring.scss

```diff
@@ -0,0 +1,51 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/vars';
+
+.instance-card {
+  background-color: #fff;
+  width: 1167px;
+  height: 210px;
+  .instance-metrics {
+    display: flex;
+    justify-content: center;
+    li {
+      padding: 20px;
+      margin: 20px;
+      text-align: center;
+      font-size: 25px;
```

Comment:
Carbon codebase uses percent unit or `rem` unit for `font-size`, so good to do the same here.